docs/data_storage.md

```diff
@@ -0,0 +1,124 @@
+# Data Storage Options
+
+## Executive Summary
+For most distributed parallel I/O-bound tasks you want to use on LUMI the stripting feature of the parallel **Lustre FS**.
+
+As far as **stripe count** goes, keep in mind that **LUMI-P has 32 OSTs** and **LUMI-F has 58 OSTs**.
+
+As fas as **stripe size** goes, safe value is **2^22 (4MiB)**, unless you know that your application uses a specific data chunk size.
+
+To configure striping with `32` stripes in `4MiB` stripe chunks all files that will be created in `/your/newly/created/data/directory`, or for an `/individual/empty/file` that you will populate, run cmd like this:
+``` bash
+lfs   setstripe   --stripe-count 32  --stripe-size 4m  /your/newly/created/data/directory   or  /individual/empty/file
+```
+
+## Intro
+
+Machine learning frameworks generally make extensive use of parallel processing to shorten time to obtain results, and so to increase the efficiency of development cycle. With increased number of nodes available for computations, a single storage target might become a bottleneck for I/O, especially when large files are to be accessed.
+
+To that end LUMI employs a parallel filesystem [Lustre](https://docs.lumi-supercomputer.eu/storage/parallel-filesystems/lustre/), which allows for storage and retrieval of files in a distributed way, using striping to distribute files across multiple storage targets (OSTs). These can be then accessed from multiple computation nodes in a parallel way, and so significantly removing said bottleneck.
+
+**Please review this [Lustre](https://docs.lumi-supercomputer.eu/storage/parallel-filesystems/lustre/) doc page to familiarize yourself with the terminology.**
+
+The main available filesystems on LUMI which have high priority for computations requiring high-throughput I/O are [LUMI-P](https://docs.lumi-supercomputer.eu/storage/parallel-filesystems/lumip/), [LUMI-F](https://docs.lumi-supercomputer.eu/storage/parallel-filesystems/lumip/) and RAM FS. In the context of massive parallel processing access, [LUMI-O](https://docs.lumi-supercomputer.eu/storage/lumio/) provides generally significantly poorer performance, and so is not considered here in this version of the guide. 
+
+To account for varying system loads, the below speeds show the averages of multiple benchmark runs. They were all performed after the LUMI system upgrade of summer 2024.
+
+To see how exactly these numbers were obtained or if you want to validate these numbers under current system load or configuration, you can run the tests yourself by downloading the SLURM batch script [here](./LUMI-lustre-fs-benchmarks-set.sh). You will need 128GB of available quota in your /scratch/project_X/ and 128GB in /flash/project_X/.
+
+## Striping
```

Comment:
This section about striping is getting very technical very quickly in my view. Could we start with providing some values for the default striping parameters for LUMI-P, LUMI-F and RAM FS? I believe most people are mostly interested in that and will only read further if they identified this as their bottleneck.